package.json

```diff
@@ -20,6 +20,7 @@
     "globby": "^11.0.1",
     "js-yaml": "^3.14.0",
     "lodash.clonedeep": "^4.5.0",
+    "needle": "^2.8.0",
```

Comment:
Can we use `cross-fetch` (Fetch API) instead of `needle` since it's already included?

Reply:
Done